android/README.md

```diff
@@ -414,50 +376,27 @@ This is a static method.
 `JitsiMeetViewListener` provides an interface apps can implement to listen to
 the state of the Jitsi Meet conference displayed in a `JitsiMeetView`.
 
-`JitsiMeetViewAdapter`, a default implementation of the
-`JitsiMeetViewListener` interface is also provided. Apps may extend the class
-instead of implementing the interface in order to minimize boilerplate.
-
-##### onConferenceFailed
-
-Called when a joining a conference was unsuccessful or when there was an error
-while in a conference.
-
-The `data` `Map` contains an "error" key describing the error and a "url" key
-with the conference URL.
-
 #### onConferenceJoined
 
 Called when a conference was joined.
 
 The `data` `Map` contains a "url" key with the conference URL.
 
-#### onConferenceLeft
+#### onConferenceTerminated
 
-Called when a conference was left.
+Called when a conference was terminated either by user choice or due to a
+failure.
 
-The `data` `Map` contains a "url" key with the conference URL.
+The `data` `Map` contains an "error" key with the error and a "url" key
+with the conference URL. If the conference finished gracefully no `error`
+key will be present.
 
 #### onConferenceWillJoin
```

Comment:
I find this one useful for the following scenario: the app launches a controller with a JitsiMeetView. It may decide to show a custom loading screen until the "will join" event. WDYT?

Reply:
Hmm but in that case the app can show custom loading just before submitting the options and wait for the conference joined. Doesn't it ?

Reply:
It's not very ergonomic, because you typically start a controller and it takes over, so being able to do this in-place will be better for the developer IMHO. We can always re-evaluate afterwards, of course.